src/cryptocontextfactory.cpp

```diff
@@ -5,13 +5,10 @@
     #undef JLCXX_HAS_RANGES
 #endif
 
-#include "jlcxx/stl.hpp"
 #include "openfhe.h"
-
```

Comment:
We usually keep this empty line to distinguish external and internal header files